docs/en/setup/examples/cases/profiling-with-vm-process/readme.md

```diff
@@ -0,0 +1,19 @@
+# Profiling with Linux VM Process
+
+This documentation helps you set up the configuration to find which VM processes want to be monitored and profiled.
+
+## Configuration
+
+### VM Process Detector
+
+You need to follow these step to configure the VM process detector:
+1. Active the VM Process Detector.
+2. Update the finder configuration.
+
+You could be following [this configuration](../../../configuration/process_discovery/vm.md) to get more configuration information.
+
+### Enable Profiling
+
+Make sure the profiling module has been active.
+
+You could be following [this configuration](../../../configuration/profiling.md) to get more configuration information.
```

Comment:
Please add a real example, such as providing a golang/C service running, and how to identify it.

Reply:
Added, please recheck.